sources/dicom/large_image_source_dicom/assetstore/rest.py

```diff
@@ -51,20 +52,19 @@ def _importData(self, assetstore, params):
             )
 
     @access.admin
-    @loadmodel(model='assetstore')
+    @boundHandler
```

Comment:
I don't think we want the `@boundHandler` decorator here.   Rather, with the changes made, I think changing the `@describeRoute` to `@autoDescribeRoute` is needed.

And, as long as we are in here, we should add a scope to the access decorator (e.g., change `@access.admin` to `@access.admin(scope=TokenScope.DATA_WRITE)`).